README.md

```diff
@@ -78,27 +78,32 @@ However, many IDEs can be configured to auto format on save.
 
 ## Proposed Reimbursement Work Flow
 
-The proposed workflow is to be automated and run on a schedule of once per _accounting
-period_ at N hours past the end of the accounting period. Note that we must wait some
-time after the period has ended for some data to finalize (e.g. `dex.trades`
-, `ethereum.transactions` our event data, etc...).
+The solver reimbursements are to be executed each Tuesday with the accounting period of the last 7 days. If the payout can not be done on Tuesdays, its okay to execute them later, but still the accounting period should always be from 00:00 between Monday and Tuesday to 00:00 between Monday and Tuesday
 
-1. Run the Reimbursement & Rewards Query
+In order to do the payout, run the following scripts:
+```
+source .env
+rm -r out/
+python -m src.fetch.transfer_file --start '2022-MM-DD' --end '2022-MM-DD'
+```
+Here, the start should be the date of the Tuesday from last week and the end day should be the Tuesday from this week.
+I.e. for the first payout, we would run:
+`python -m src.fetch.transfer_file --start '2022-03-01' --end '2022-03-08' `
```

Comment:
If this is the example, then your description of monday-tuesday above is incorrect.

The results of this will be from beginning march 1 (a monday) to end of monday the next week (a total of 8 days). 

According to the description above, this should be 
```suggestion
`python -m src.fetch.transfer_file --start '2022-03-02' --end '2022-03-08'`
```

Reply:
Sorry - forget that. For some reason I thought March 1 was a Monday!

Reply:
First of March is a Tuesday in my calendar